packages/block-library/src/embed/embed-placeholder.native.js

```diff
@@ -40,12 +44,58 @@ const EmbedPlaceholder = ( {
 	);
 	const embedIconErrorStyle = styles[ 'embed__icon--error' ];
 
+	const cannotEmbedMenuPickerRef = useRef();
+
+	const errorPickerOptions = {
+		retry: {
+			id: 'retryOption',
+			label: __( 'Retry' ),
+			value: 'retryOption',
+			onSelect: () => {
+				tryAgain();
+			},
+		},
+		convertToLink: {
+			id: 'convertToLinkOption',
+			label: __( 'Convert to link' ),
+			value: 'convertToLinkOption',
+			onSelect: () => {
+				fallback();
+			},
+		},
+	};
+
+	const options = compact( [
+		cannotEmbed && errorPickerOptions.retry,
+		cannotEmbed && errorPickerOptions.convertToLink,
+	] );
+
+	function onPickerSelect( value ) {
+		const selectedItem = options.find( ( item ) => item.value === value );
+		selectedItem.onSelect();
+	}
+
+	function onPickerPresent() {
+		if ( cannotEmbedMenuPickerRef.current ) {
+			cannotEmbedMenuPickerRef.current.presentPicker();
+		}
+	}
+
+	// when the content cannot be embedded the onPress should trigger the Picker instead of the onPress prop.
```

Comment:
We usually capitalize the first letter of a comment so we could change this to:
```suggestion
	// When the content cannot be embedded the onPress should trigger the Picker instead of the onPress prop.
```

Reply:
Done here deb2eb7